src/Illuminate/Database/Eloquent/Concerns/QueriesRelationships.php

```diff
@@ -188,7 +188,74 @@ public function orWhereDoesntHave($relation, Closure $callback = null)
      * @param  mixed  $relations
      * @return $this
      */
-    public function withCount($relations)
+    public function withCount(...$relations)
+    {
+        return $this->withAggregate('count', ...$relations);
+    }
+
+    /**
+     * Add subselect queries to retrieve the minimum value of a column on the relations.
+     *
+     * @param  mixed  $relations
```

Comment:
All these params should probably be `array` eh?

Reply:
Not necessarily. Examples of valid calls:
`->withCount('posts')`
`->withCount('posts', 'tags')`
`->withCount(['posts' => function ($query) { ... }, 'tags' => function ($query) { ... }])`